lib/cadet_web/controllers/assessments_controller.ex

```diff
@@ -240,6 +243,13 @@ defmodule CadetWeb.AssessmentsController do
               "The external library for this question"
             )
           end
+        end,
+      GraderInfo:
```

Comment:
You only need to define this once to keep things DRY (Don't Repeat Yourself). Perhaps the best place to put it is in the `grading_controller.ex`, so just remove this instance